### ISSUE TITLE
Installation fails on Ubuntu, Python 2.7

### ISSUE THREAD

**Issue by stephane-caron** [cvxopt/cvxopt#121]
Installation by ``pip install cvxopt`` fails as follows on my system (Ubuntu 14.04, Python 2.7.6):
```
building 'umfpack' extension

x86_64-linux-gnu-gcc -pthread -fno-strict-aliasing -DNDEBUG -g -fwrapv -O2 -Wall -Wstrict-prototypes -fPIC -I/usr/include/suitesparse -I/usr/include/python2.7 -c src/C/umfpack.c -o build/temp.linux-x86_64-2.7/src/C/umfpack.o

[... just a warning warning ...]

x86_64-linux-gnu-gcc -pthread -shared -Wl,-O1 -Wl,-Bsymbolic-functions -Wl,-Bsymbolic-functions -Wl,-z,relro -fno-strict-aliasing -DNDEBUG -g -fwrapv -O2 -Wall -Wstrict-prototypes -D_FORTIFY_SOURCE=2 -g -fstack-protector --param=ssp-buffer-size=4 -Wformat -Werror=format-security build/temp.linux-x86_64-2.7/src/C/umfpack.o -L/usr/lib/x86_64-linux-gnu -lumfpack -lcholmod -lamd -lcolamd -lsuitesparseconfig -llapack -lblas -lrt -o build/lib.linux-x86_64-2.7/cvxopt/umfpack.so

/usr/bin/ld: /usr/lib/x86_64-linux-gnu/libsuitesparseconfig.a(SuiteSparse_config.o): relocation R_X86_64_PC32 against undefined symbol `malloc@@GLIBC_2.2.5' can not be used when making a shared object; recompile with -fPIC

/usr/bin/ld: final link failed: Bad value

collect2: error: ld returned 1 exit status

error: command 'x86_64-linux-gnu-gcc' failed with exit status 1
```
This only happens for ``cvxopt==1.2.0``.

For reference, on the same system:
- ``pip install cvxopt==1.1.9`` fails with the error "umfpack.h: No such file or directory"
- ``pip install cvxopt==1.1.8`` works just fine

**Comment by martinandersen** [comment]
Our precompiled 'manylinux' wheels work on most Linux systems, but it appears that your system is incompatible. Starting from version 1.1.9 you have to provide SuiteSparse as a shared library or the source when building CVXOPT from source. Here's an excerpt from the [documentation](http://cvxopt.org/install/index.html#ubuntu-debian):
```
wget http://faculty.cse.tamu.edu/davis/SuiteSparse/SuiteSparse-5.2.0.tar.gz
tar -xf SuiteSparse-5.2.0.tar.gz
export CVXOPT_SUITESPARSE_SRC_DIR=$(pwd)/SuiteSparse
pip install cvxopt
```

**Comment by stephane-caron** [comment]
Thanks! This fixes the installation for both 1.1.9 and 1.2.0 on Ubuntu 14.04 (although with 1.2.0 I ran into another issue #123...).
